apps/angular/src/app/features/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,44 @@
+import { AsyncPipe, DatePipe } from '@angular/common';
+import { Component, inject } from '@angular/core';
+import { MatTableModule } from '@angular/material/table';
+import { TableColumn } from '@js-camp/angular/core/models/table-column';
+import { EmptyPipe } from '@js-camp/angular/core/pipes/empty.pipe';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+
+/** Column key values. */
+enum ColumnKey {
+	Image = 'image',
+	TitleEng = 'titleEng',
+	TitleJpn = 'titleJpn',
+	AiredStart = 'airedStart',
+	Type = 'type',
+	Status = 'status',
+}
+
+/** Anime table component. */
+@Component({
+	selector: 'anime-table',
+	templateUrl: './anime-table.component.html',
+	styleUrl: './anime-table.component.css',
+	standalone: true,
+	imports: [MatTableModule, AsyncPipe, DatePipe, EmptyPipe],
+})
+export class AnimeTableComponent {
+	private readonly animeService = inject(AnimeService);
+
+	/** Represents anime list. */
+	protected readonly anime$ = this.animeService.getAll();
+
+	/** Represents table columns. */
+	protected readonly displayedColumns: readonly TableColumn<ColumnKey>[] = [
+		{ key: ColumnKey.Image, header: 'Image' },
+		{ key: ColumnKey.TitleEng, header: 'English title' },
+		{ key: ColumnKey.TitleJpn, header: 'Japanese title' },
+		{ key: ColumnKey.AiredStart, header: 'Aired start' },
+		{ key: ColumnKey.Type, header: 'Type' },
+		{ key: ColumnKey.Status, header: 'Status' },
+	];
+
+	/** Header row definition. */
+	protected readonly headerRowDef = this.displayedColumns.map(column => column.key);
```

Comment:
Please don't use abbreviations other than the generally accepted ones (for example, DTO). If I had seen this field in the template, I would hardly have understood what it means and would have had to go to the component to read the comment.

```suggestion
	protected readonly headerRowDefinitions = this.displayedColumns.map(column => column.key);
```